modules/oe_translation_cdt/src/CdtLanguageMapper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\oe_translation_cdt;
+
+/**
+ * Maps the Drupal language to the configured CDT language.
+ */
+class CdtLanguageMapper {
```

Comment:
I've created this class similar to the ePoetry module.

Now that I'm developing D8AGE-290, I've researched some best approaches in Entity <=> DTO conversion, and it seems that Symfony-based projects should define dedicated mapper classes (right now I'm working on `TranslationRequestMapper`, which assembles a DTO from TranslationRequest entity). 

So the idea is good, but maybe we should create a `Mapper` directory and put all related mapper classes there.

Just putting this out there for your consideration.

Reply:
sounds good to me